doc/md/fundamentals/2-types/3-functions.md

```diff
@@ -296,6 +303,8 @@ Multiple values can be passed as a single argument by encapsulating them within
   public func userName(user: { name : Text; age : Nat }) : Text {
     user.name;
   }
+```
+
```

Comment:
This function can also be written using a simpler record pattern, that matches on the fields of the record instead.
```motoko no-repl
func userName({ name : Text; age : Nat }) : Text {
  name
}
```